src/main/java/org/alliancegenome/curation_api/services/validation/dto/fms/GeneExpressionAnnotationFmsDTOValidator.java

```diff
@@ -127,7 +113,7 @@ public GeneExpressionAnnotation validateAnnotation(GeneExpressionFmsDTO geneExpr
 
 		ObjectResponse<TemporalContext> temporalContextObjectResponse = validateTemporalContext(geneExpressionFmsDTO);
 		if (temporalContextObjectResponse.hasErrors()) {
-			response.addErrorMessage("whenExpressed", temporalContextObjectResponse.errorMessagesString());
+			response.addErrorMessage("expressionPattern", temporalContextObjectResponse.errorMessagesString());
```

Comment:
This change also doesn't make sense to me

Reply:
With that change the errors will be like "expressionPattern - whenExpressed - error message"